test/api.test.ts

```diff
@@ -21,6 +21,18 @@ describe("api (workspace)", () => {
         expect(installDependenciesSpy).toHaveReturned();
       }, 60_000);
 
+      it("installs dependencies with lockfile", async () => {
```

Comment:
I considered adding a more substantial test where I parse out the installed version after installing with the lockfile and ensure a version bump wasn't done. I wasn't able to get `npm` or `pnpm` to reliably bump the version, so I resorted to this test.

LMK if you see a better test.